src/yz_exchange_fsm.erl

```diff
@@ -163,21 +163,17 @@ key_exchange(timeout, S=#state{index=Index,
 
     AccFun = fun(KeyDiff, Count) ->
                      lists:foldl(fun(Diff, InnerCount) ->
-                                         case repair(Index, Diff) of
-                                             full_repair -> InnerCount + 1;
-                                             _ -> InnerCount
-                                         end
+                                     case repair(Index, Diff) of
+                                         full_repair -> InnerCount + 1;
+                                         _ -> InnerCount
+                                     end
                                  end, Count, KeyDiff)
              end,
-
     case yz_index_hashtree:compare(IndexN, Remote, AccFun, 0, YZTree) of
         0 ->
-            yz_kv:update_aae_exchange_stats(Index, IndexN, 0),
-            ok;
+            yz_kv:update_aae_exchange_stats(Index, IndexN, 0);
         Count ->
-            yz_kv:update_aae_exchange_stats(Index, IndexN, Count),
-            lager:info("Repaired ~b keys during active anti-entropy exchange "
-                       "of partition ~p for preflist ~p",
+            lager:info("Will repair ~b keys of partition ~p for preflist ~p",
```

Comment:
As per discussion w/ @fadushin, we may want to change the wording of this/handle-this better, as it's the *to*-repair count vs actual repairs when the related operations go through batching.